onnx/defs/quantization/defs.cc

```diff
@@ -54,27 +62,30 @@ ONNX_OPERATOR_SET_SCHEMA(
               updateOutputShape(ctx, 0, input_shape);
         }));
 
-static const char* DequantizeLinear_ver10_doc = R"DOC(
-The linear dequantization operator. It consumes a quantized tensor, a scale, a zero point to compute the full precision tensor.
-The dequantization formula is y = (x - x_zero_point) * x_scale. 'x_scale' and 'x_zero_point' must have same shape.
+static const char* DequantizeLinear_ver13_doc = R"DOC(
+The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full precision tensor.
+The dequantization formula is y = (x - x_zero_point) * x_scale. 'x_scale' and 'x_zero_point' must have same shape, and can be either a scalar
+for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantizations.
 'x_zero_point' and 'x' must have same type. 'x' and 'y' must have same shape. In the case of dequantizing int32,
 there's no zero point (zero point is supposed to be 0).
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     DequantizeLinear,
-    10,
+    13,
     OpSchema()
         .Input(0, "x", "N-D quantized input tensor to be de-quantized.", "T")
         .Input(
             1,
             "x_scale",
-            "Scale for input 'x'. It's a scalar, which means a per-tensor/layer quantization.",
+            "Scale for input 'x'. It can be a scalar, which means a per-tensor/layer dequantization, "
+            "or a 1-D tensor for per-axis dequantization.",
             "tensor(float)")
         .Input(
             2,
             "x_zero_point",
-            "Zero point for input 'x'. It's a scalar, which means a per-tensor/layer quantization. "
+            "Zero point for input 'x'. It can be a scalar, which means a per-tensor/layer dequantization, "
+            "or a 1-D tensor of size C for per-channel dequantization. "
```

Comment:
description not consistent. line 82 mentions per-axis dequantization and line 88 says per-channel dequantization

Reply:
Good catch, fixed.